code/datums/helper_datums/teleport.dm

```diff
@@ -194,21 +194,23 @@
 			teleatom.visible_message("<span class='danger'>The Bag of Holding bounces off of the portal!</span>")
 			return 0
 
-	if(istype(teleatom,/obj/item/clothing/head/tinfoil))
+	if(destination.z > 7) //Away mission z-levels
```

Comment:
Oh.
Well do it anyways, I don't want some chump making a 7-z map in the future and raising a bug report.